src/lib/containers/entity/annotations/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -298,25 +297,29 @@ export const SchemaDrivenAnnotationEditor = (
                 Annotations could not be updated: {submissionError.reason}
               </Alert>
             )}
-            <hr />
-            <div className="SaveButtonContainer">
-              <Button
-                variant="primary-500"
-                onClick={() => {
-                  formRef.current!.submit()
-                }}
-              >
-                {entityId ? 'Save' : 'Validate'}
-              </Button>
-              {onCancel && (
-                <>
-                  <div className="Spacer" />
-                  <Button variant="primary-500" onClick={onCancel}>
-                    Cancel
+            {!formRefFromParent && (
```

Comment:
If we don't have a ref passed in, then render this block, which includes save/cancel buttons.

If we do have a ref passed in, then don't render it, because the parent is handling submission/cancellation